src/test/java/com/github/cameltooling/lsp/internal/completion/CamelKubernetesServicesCompletionTest.java

```diff
@@ -39,13 +44,19 @@
 @EnableKubernetesMockClient(crud = true)
 class CamelKubernetesServicesCompletionTest extends AbstractCamelLanguageServerTest {
 
+	@Inject
 	private KubernetesClient client;
-	
+
+	@AfterEach
+	void cleanCluster() {
+		KubernetesConfigManager.getInstance().setClient(client);
+		client.namespaces().delete();
+		client.secrets().delete();
```

Comment:
this part is `cleanCluster` and has its place in @AfterEach

Reply:
On AfterEach the client is closed, again. Looking for a work around.

Reply:
Looks like cleanup is no longer needed? What the...?

Reply:
It was not needed on my local only, because whatever :)

But if I try to move those deletes to the `AfterEach`, the kubernetes client is closed by then